README.md

```diff
@@ -1,20 +1,20 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Leaderboard
 
-> One paragraph statement about the project.
+> In this activity I set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules. I consume the Leaderboard API using JavaScript async and await and add some styling.
 
 ![screenshot](./app_screenshot.png)
```

Comment:
- [ ] Please add a screenshot of your project or remove this line to keep it professional. As it is right now it loads a broken link. Remember that at any point recruiters might visit your recent projects in GitHub and that they're professionally documented is our responsability.